### PR TITLE
Seeding script for tracks db

### PR DESCRIPTION
**Pull request by alephno**

### CODE REVIEW

**Comment by mille424** [comment]
Looks good, cool use of toString(36).

**Comment by alephno** [comment]
> Looks pretty good, I just don't see track art.

Just added it in as art_url

File: db/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+mongoose.connect('mongodb://localhost/tracks', { useNewUrlParser: true })
+
+const TrackSchema = new mongoose.Schema({
+  name: String,
+  artist: String,
+  cdn_url: String,
+  tags: [String],
+  plays: { type: Number, default: 0},
+  likes: { type: Number, default: 0},
+  reposts: { type: Number, default: 0},
```
Comment (LesliePajuelo) [inline db/index.js]: Good use of defaults

File: db/seed.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      artist: 'AmigoKing',
+      cdn_url: `http://www.whatever.com/${randomString()}.mp3`,
+      art_url: `http://www.whatever.com/${randomString()}.jpg`,
+      tags: randomTags(),
+      plays: randomNum(100000),
+      likes: randomNum(1000),
+      reposts: randomNum(100)
+  }))
+
+  Promise.all(tracks).then((results) => {
```
Comment (LesliePajuelo) [inline db/seed.js]: Are you only using bluebird for all()?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

**Pull request merged**